### DISCUSSION TITLE
Is this official 1-Click app for Digital Ocean?

### DISCUSSION THREAD

**Discussion by vigneshksaithal** [pocketbase/pocketbase#1486] [Q&A] [answered]
Hi, is this the official marketplace app for deploying PocketBase in Digital Ocean? If Yes, how to update it?

Link to app: https://marketplace.digitalocean.com/apps/pocketbase

[NOT RELATED] I love PocketBase more than anything. And I salute the creator and contributors. You people are really motivational and I love building with PocketBase.

**Comment by ganigeorgiev** [comment, marked as answer]
No, this is not an official image. I can't find information about the image and what actually it contains so I cannot recommend it.

To update PocketBase you usually have to replace the binary, but I'm not sure where it is located in this case.

_**Update**: See the answers below._

**Comment by solocreativeco** [comment]
I used that image on digital ocean. The files live in /opt/pocketbase/pb_data. I had to SCP my local pb_data and pb_migration folders so I would not have to rebuild everything from scratch. Using that image for proof of concept only. For production I plan to create my own.

**Comment by scott** [comment]
Hey Guys- I am a huge fan of PocketBase and decided to create and maintain this one-click for DO.  I work for DO and am a marketplace vendor as well.  The Packer build script is here - https://github.com/scott/pocketbase-do-packer.  Please feel free to submit PRs and suggest improvements to the layout.  I would also gladly transfer any of this to the official PocketBase org.

I meant to announce this into the "show and tell" topic last week but forgot to before i went out of town for the new years holiday.

**Reply by ganigeorgiev** [reply]
Thank you for sharing the packer script (_I've contacted earlier the DO support to provide some info; I'll update the ticket shortly_).

If possible, it would be nice to add the link to the packer repo listed somewhere (in the listing description or in the sidebar?) so that users can inspect and check what will be installed before running the app.

**Reply by scott** [reply]
Got it already although it could take a few minutes for the caching to update!  Super excited about your product and being able to contribute the build script 😄

**Reply by vigneshksaithal** [reply]
@scott Could you please make this official 1-Click app on DO? Would be very helpful for beginners like me.
